app/javascript/pages/account/FormSubmitButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "@bloom-housing/ui-components"
+import React from "react"
+
+interface FormSubmitButtonProps {
+  loading: boolean
+  label: string
+}
+
+const FormSubmitButton = ({ loading, label }: FormSubmitButtonProps) => {
+  return (
+    <div className="flex justify-center">
+      <Button loading={loading} type="submit">
```

Comment:
Can you switch this to use the Seeds version of Button?

Reply:
The Seeds version doesn't take a loading prop

Reply:
Ah got it, sorry I missed that!